src/Entity/CommitteeMembership.php

```diff
@@ -26,6 +27,10 @@
  *     }
  * )
  * @ORM\Entity(repositoryClass="AppBundle\Repository\CommitteeMembershipRepository")
+ * @UniqueEntity(
```

Comment:
This check is done on the ORM level.
The database will still allow multiple `enableVote=1` for the same adherent.

you should add a @ORM\UniqueConstraint also (like the one already setup to ensure the same adherent does not join the same committee twice)

see https://github.com/EnMarche/en-marche.fr/blob/2655b16a46084c57891fb6abedfcfb3960d7ac7f/src/Entity/CommitteeMembership.php#L19-L24

Reply:
Can't, otherwise you'll only allow adherent to join 1 committee without voting there, and one with vote rights.

Reply:
When adding a unique constraint in SQL, `NULL` values are not taken into account.

If you add a Unique constraint on the ORM level (with the @UniqueEntity annotation), then you should also add a Unique constraint on the database level.

Just add
```

  *     uniqueConstraints={ 
  *          [...],
  *         @ORM\UniqueConstraint( 
  *             name="adherent_votes_in_committee", 
  *             columns={"adherent_id", "enable_vote"} 
  *         ) 
  *     }, 

```

to ensure this unique constraint is also added on the database level.

One adherent can only vote for one committee.